### PR TITLE
[Seller selector] Fix for invited sellers unable to apply for opportunities

### PR DESCRIPTION
**Pull request by tobiasbrown**
Fixes MAR-1910

Related PR:
- [supplier front-end](https://github.com/AusDTO/dto-digitalmarketplace-supplier-frontend/pull/274)

### CODE REVIEW

File: app/main/views/marketplace.py [hunk 1]
```diff
@@ -108,6 +108,7 @@ def get_brief_by_id(framework_slug, brief_id):
         brief_responses = data_api_client.find_brief_responses(
             brief_id, current_user.supplier_code)["briefResponses"]
         selected_for_brief = _is_supplier_selected_for_brief(brief)
+        print 'selected_for_brief: ', selected_for_brief
```
Comment (malmas18) [inline app/main/views/marketplace.py]: remove print

File: app/main/views/marketplace.py [hunk 2]
```diff
@@ -88,11 +88,11 @@ def domain(email):
     if brief.get('sellerSelector', '') == 'someSellers':
         seller_domain_list = [domain(x).lower() for x in brief.get('sellerEmailList', [])]
         return current_user.email_address in brief.get('sellerEmailList', []) \
```
Comment (malmas18) [inline app/main/views/marketplace.py]: should this also do a comparison on lower case?

Comment (tobiasbrown) [reply app/main/views/marketplace.py]: Added this for consistency.

File: app/main/views/marketplace.py (hunk 1 above)
Comment (tobiasbrown) [reply app/main/views/marketplace.py]: Cleaned this up.

**Pull request merged**